### PR TITLE
Fix deprecation

### PR DESCRIPTION
**Pull request by jumagura**

### CODE REVIEW

**Comment by merefield** [comment]
Looks good.  However, is it worth adding front end tests just on this scope as a start to incrementally increasing coverage?

**Comment by jumagura** [comment]
HI @merefield just updating this PR for bootbox deprecation. I will create another PR for frontend tests and CI. if that is okay with you

**Comment by merefield** [comment]
Ooh thanks for this @jumagura.  If you don't mind I will review on Monday?

**Review by merefield — CHANGES_REQUESTED** [review]

File: assets/javascripts/discourse/templates/components/rating-migrate.hbs [hunk 1]
```diff
@@ -20,10 +20,10 @@
     none="admin.ratings.type.select"
   )}}
 
-{{d-button
-  action="migrate"
-  label="admin.ratings.migrate.btn"
-  disabled=migrateDisabled}}
+<DButton
+  @action={{action "migrate"}}
```
Comment (merefield) [inline assets/javascripts/discourse/templates/components/rating-migrate.hbs:24]: This should reference the action using `this` not using a string.

File: assets/javascripts/discourse/templates/components/rating-destroy.hbs [hunk 2]
```diff
@@ -11,10 +11,11 @@
     none="admin.ratings.type.select"
   )}}
 
-{{d-button
-  action="destroyRatings"
-  label="admin.ratings.destroy.btn"
-  disabled=destroyDisabled}}
+<DButton
+  @action={{action "destroyRatings"}}
```
Comment (merefield) [inline assets/javascripts/discourse/templates/components/rating-destroy.hbs:15]: This should reference the action using `this` not using a string.

**Review by merefield — CHANGES_REQUESTED** [review]

**Review by merefield — APPROVED** [review]

**Comment by merefield** [comment]
thanks @jumagura ! 👍

**Pull request merged**